SHP.OnlineShopAPI.Tests/ControllerTests/CategoryControllerTests.cs

```diff
@@ -7,15 +7,15 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Moq;
-using OnlineShopAPI.Controllers;
+using SHP.OnlineShopAPI.Controllers;
 using OnlineShopAPI.DTO.Category;
 using OnlineShopAPI.Mapping;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Xunit;
 
-namespace OnlineShopAPI.Tests
+namespace SHP.OnlineShopAPI.Tests
```

Comment:
Some changes needed in Tests namespaces, for example, SHP.OnlineShopAPI.Tests -> SHP.OnlineShopAPI.Tests.ControllerTests

Reply:
Fixed